games/u.yaml

```
@@ -1,3 +1,24 @@
+- name: UZDoom
+  langs:
+  - C++
+  frameworks:
+  - OpenGL
+  - Vulkan
+  licenses:
+  - GPL3
+  development: very active
+  originals:
+  - Doom
+  - Heretic
+  - Hexen
+  status: unplayable
```

```suggestion
  status: playable
```
it is a fork so it should be playable already?